photo-editor/dom.js

```diff
@@ -0,0 +1,12 @@
+function editPhoto() {
+    const filters = document.querySelectorAll('input');
+    
+    filters.forEach(filter => filter.addEventListener('change', filtersUpdate));
+    filters.forEach(filter => filter.addEventListener('mousemove', filtersUpdate));
```

Comment:
Let's have one event listener here instead of one for each input. and think about how we can change filtersUpdate function in order to fit out new approach